spec/xcpretty/parser_spec.rb

```diff
@@ -28,6 +28,11 @@ module XCPretty
       @parser.parse(SAMPLE_BUILD)
     end
 
+    it "parses analyze target" do
+      @formatter.should receive(:format_analyze_target).with("The Spacer", "Pods", "Debug")
```

Comment:
Line is too long. [91/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.